docs/concepts/codesharing.md

```diff
@@ -11,23 +11,23 @@ ms.technology: .net-core-technologies
 ms.devlang: dotnet
 ms.assetid: d4c31c29-e977-48f2-b4c7-0335ef50d440
 ---
+
 # Code Sharing with .NET Core and the .NET Standard Library #
 
 The .NET Standard Library aims at maintaining binary compatibility between the .NET-capable platforms for which an application is developed and future versions of those platforms. It also supports source code compatibility between platforms that support a particular version of the .NET Standard.
 
-<aside style="border-style:double">
-**Important:** This is very preliminary documentation for the .NET Standard Library. Because the standard has not been formalized yet, it is subject to change.
-</aside>
+[!IMPORTANT]
+This is very preliminary documentation for the .NET Standard Library. Because the standard has not been formalized yet, it is subject to change.
```

Comment:
I believe the correct syntax requires `>`s:

```md
> [!IMPORTANT]
> This is very preliminary documentation for the .NET Standard Library. Because the standard has not been formalized yet, it is subject to change.
```